ui/shared/status/StatusChatInput.qml

```diff
@@ -1065,6 +1105,7 @@ Rectangle {
                 type: "secondary"
                 onClicked: {
                     emojiPopup.close()
+                    gifPopup.close()
```

Comment:
I don't remember why I didn't do it before, but all those `close` calls could be put in a function I think. Just need to pass which one you want to open and close the others, instead of copying that code for each button. Again, that's my bad

Reply:
Updated, should be better now ;) good suggestion